operator/bucketcontroller/connector.go

```diff
@@ -24,9 +25,10 @@ type bucketConnector struct {
 	kube     client.Client
 	recorder event.Recorder
 
-	bucket            *cloudscalev1.Bucket
-	credentialsSecret *corev1.Secret
-	minio             *minio.Client
+	bucket                            *cloudscalev1.Bucket
+	credentialsSecret                 *corev1.Secret
+	minio                             *minio.Client
+	credentialSecretMissingAsExpected bool
```

Comment:
maybe we can rename it `bucketAlreadyDeleted`? It's not as descriptive as yours but it's more straight forward.

Reply:
this is already outdated